### ISSUE TITLE
A11y_OSvisualupdate_MiniControGallery_GenericControls_Screenreader : The table headers for the calendar were not announced for screen reader users.

### ISSUE THREAD

**Issue by MayuriNaik** [microsoft/microsoft-ui-xaml#1588]
GitHubTags: #A11y_OSvisualupdate, #A11yMAS, #A11yTCS, #Win10, #DesktopApp, #WindowsUILibrary, #MAS4.2.3, #NVDA, #Narrator

**Environment Details:**
Application Name: OS visual update
Windows Version: Windows10
Screen Reader: Narrator, NVDA 2019.2

**Steps to Reproduce:**
1. Launch Mini Control Gallery App from the installer location.
2. Screen with all generic controls would appear on left side of the screen.
3. Verify that all the elements of "MiniControlGallery-Generic Controls" are accessible and MAS Compliant.

**Actual:**
The calendar view is announced as with table structure but the “Sun”, “Mon”, “Tue”, etc. are not announced as table headers for the screen reader users.

**Similar issue is observed in the below scenario:**
1. MiniControlGallery - Calender and Color Picker Controls -> Launch Mini Control Gallery App from the installer location. -> Screen with Calender and Color Picker controls would appear on right side of the screen.

**Expected:**
The “Sun”, “Mon”, “Tue”, etc. should be announced as table headers to the screen reader users.

**User Impact:**
If proper information is not announced to the screen reader users, then they will get confused and might miss out on the information.

**Attachment:**
[A11y_OSvisualupdate_MiniControGallery_GenericControls_Screenreader  The table headers for the calendar were not announced for screen reader users..pptx](https://github.com/microsoft/microsoft-ui-xaml/files/3830673/A11y_ControlUIupdatebasedonSantoriniOSvisualupdate_MiniControGallery_GenericControls_Screenreader.The.table.headers.for.the.calendar.were.not.announced.for.screen.reader.users.pptx)

**Comment by jevansaks** [comment]
This seems to be a narrator choice. The UIA elements have this information provided but I think the default narrator read-out must be excluding it. If you increase the verbosity (with Caps+V) to, say, level 5, then the headers do get read out. This is working as intended.
